### PR TITLE
Add Bruno

### LINKED ISSUE #2520: Move API code examples inside codebase
## User story

As a developer I want an easy way to test the API, and I want the API examples to remain inside the codebase so they stay up to date.

## Acceptance criteria

- [ ] API examples from the handbook are in the codebase
- [ ] Requests generated by [Trans Dimension are documented](https://github.com/GenderedIntelligence/the-trans-dimension/tree/main/src/Data/PlaceCal) and have minimal testing
- [ ] README updated to show how to run them
- [ ] Handbook updated to point to the Bruno examples

## Implementation notes & questions

Plan is to use [Bruno](https://www.usebruno.com/) for this.

## Implementation plan

_To be written by the developer_

### PR DESCRIPTION
**Pull request by kimadactyl**
AC need some work but just wanted to try this out. It looks super neat and is wayyy more usable than curl for debugging.

It's showing up the gaps in my knowledge with GraphQL tho, especially with pagination.

Fixes #2520

### CODE REVIEW

**Comment by kimadactyl** [comment]
Oh this is no way ready for merge yet - we need to have some AC first and flesh it out. More a WIP to think about esp as you do the changes to TransDim.

File: collections/Get Event.bru [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      description
+      startDate
+      endDate
+      address {
+        streetAddress
+        postalCode
+        addressLocality
+        addressRegion
+      }
+      organizer {
```
[the remark this answers is not in the document]
Comment (kimadactyl) [reply collections/Get Event.bru:28]: yes, we matched schema.org spec

File: collections/Get Event.bru [hunk 2]
```diff
@@ -0,0 +1,35 @@
+meta {
+  name: Get Event
```
[the remark this answers is not in the document]
Comment (kimadactyl) [reply collections/Get Event.bru:2]: yep

File: collections/Get Events.bru [hunk 3]
```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Get Events
+  type: graphql
+  seq: 5
+}
+
+post {
+  url:
```
[the remark this answers is not in the document]
Comment (kimadactyl) [reply collections/Get Events.bru:8]: yeah this is really far from being able to merge haha

**Comment by katjam** [comment]
Phew. Glad you are on the same page. :)

**Comment by kimadactyl** [comment]
Probably a really good thing you do next time you work on TransDim is to put the exact GraphQL queries it uses into Bruno so we can then have documentation of what TransDim is doing and maybe even put some tests in against it?

**Comment by katjam** [comment]
> Probably a really good thing you do next time you work on TransDim is to put the exact GraphQL queries it uses into Bruno so we can then have documentation of what TransDim is doing and maybe even put some tests in against it?

Great idea.

**Comment by kimadactyl** [comment]
Superceded by #2535

**Pull request closed without merging**